js/main.js

```diff
@@ -0,0 +1,112 @@
+const featureGuest = {
+  guestOne: {
+    name: 'Luc Mayitoukoua',
+    job: 'Director ZHU Culture from Senegal',
+    subject: 'Markets and cultural networks',
+    picture: './img/luc.png',
+  },
+  guestTwo: {
+    name: 'Stéphanie Soleansky',
+    job: 'Consultant and cultural entrepreneur from France',
+    subject: 'Women and cultural entreprenership',
+    picture: './img/steph.png',
+  },
+  guestThree: {
+    name: 'Natalie Aloessode Bernardin',
+    job: 'Artist manager from Canade',
+    subject: 'Women and cultural entreprenership',
+    picture: './img/natalie.png',
+  },
+  guestFour: {
+    name: 'Mawuto Dick',
+    job: 'Director of the togoville jazz festival from Togo',
+    subject: 'The music charters of the world',
+    picture: './img/mawuto.png',
+  },
+  guestFive: {
+    name: 'Cyriaque Noussouglo',
+    job: 'Secretary of national committee of francophonie in togo',
+    subject: 'cultural and creative industries',
+    picture: './img/cyriaque.png',
+  },
+  guestSix: {
+    name: 'Vanessa Kanga',
+    job: 'Artist and co-founder of Afropolitain Nomade from Canada',
+    subject: 'women and cultural entreprenership',
+    picture: './img/vanessa.png',
+  },
+};
+
+const featureSpeakersSection = document.querySelector('.featured-speakers');
+
+const SpeakersSection = document.querySelector('.speakers-section');
+
+const featuredSpeakersTitle = document.createElement('h3');
+featuredSpeakersTitle.className = 'lato-font-black';
+featuredSpeakersTitle.innerHTML = 'Featured Speakers<span class="underline-style"></span>';
+SpeakersSection.append(featuredSpeakersTitle);
+
+const moreSpeakers = document.createElement('button');
+moreSpeakers.className = 'lato-font-black';
+moreSpeakers.innerHTML = 'More  ▼';
+
+SpeakersSection.append(moreSpeakers);
+
+function callSpeakers(guest) {
+  const everyFeatureDiv = document.createElement('div');
```

Comment:
- [ ] I suggest you use an `article` element instead of a `div` for the reasons discussed earlier. Please update the `font-size` and `font-style` to match the design. The orange text is in italics style on the design. 🤓️ There's also a pattern behind the image. Please update it accordingly.


| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217800772-fbd2084d-93fa-4485-9095-02fc147c624a.png" width="400">|<img src="https://user-images.githubusercontent.com/17243325/217800769-e3b7c64e-0c3d-45cd-98e2-e56f1ccbd0fe.png" width="400">|